playground/src/components/CodingEnvironment/MemoryInfo.tsx

```diff
@@ -21,7 +21,7 @@ const MemoryInfo = (props: MemoryInfoProps): JSX.Element => {
           <div className="flex flex-wrap overflow-auto h-40">
             {props.debugOutput &&
               formatMemory(props.debugOutput.memory).map(
-                (memoryString, index) => (
```

Comment:
was this a bug?

Reply:
`eslint` complained because `index` was not used